doc/vagrant/share/packelektra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Its better to execute the script than doing copy and paste.

Or do you have an idea how we can have docker/vagrant with Elektra already installed? (Both is sometimes needed)